x-pack/test/security_solution_endpoint/apps/endpoint/index.ts

```diff
@@ -13,7 +13,7 @@ import {
 export default function (providerContext: FtrProviderContext) {
   const { loadTestFile, getService } = providerContext;
 
-  describe('endpoint', function () {
+  describe.skip('endpoint', function () {
```

Comment:
Heads up @kevinlog we need to do this because of the breaking changes for `dataset` -> `data_stream`. We'll reenable as soon as the endpoint package with the same changes gets released: https://github.com/elastic/endpoint-package/pull/89

Reply:
no problem, I have PR right now where I'm re-enabling our other tests due to the one faulty package. https://github.com/elastic/kibana/pull/78699

I'm happy to to wait on it since this PR seems more complicated.  I could also re-enable everything after the package releases.

Reply:
After speaking with @jen-huang offline, it's OK to merge this PR #78954.  Tests will be disabled again with this PR, but we'll re-enable again after the next package.